accelerator/errors.h

```diff
@@ -51,9 +51,12 @@ extern "C" {
 typedef enum {
   SC_OK = 0, /**< No error */
 
-  SC_HTTP_OK = 200,          /**< HTTP response OK */
-  SC_HTTP_BAD_REQUEST = 400, /**< HTTP response, error when parsing request */
-  SC_HTTP_NOT_FOUND = 404,   /**< HTTP request not found */
+  SC_HTTP_METHOD_GET = 10,     /**< HTTP GET method */
+  SC_HTTP_METHOD_POST = 11,    /**< HTTP POST method */
+  SC_HTTP_METHOD_OPTIONS = 12, /**< HTTP OPTIONS method */
```

Comment:
What is the purpose of these status codes? 🤔  ex: 10, 11, 12

Reply:
Hundreds are already took by http status and I would like to reserve units to general purpose. So I thought decimals might fit.

Reply:
The abbreviation of "for example" is [e.g.](https://en.wiktionary.org/wiki/e.g.) rather than "ex."